doc/nrf/known_issues.rst

```diff
@@ -3035,6 +3035,14 @@ Trusted Firmware-M (TF-M)
 
 .. rst-class:: v2-3-0
```

Comment:
Probably affects more versions, hard to know here really.

Reply:
Agree, so is your suggestion to try to find the exact version? Or just try 2-3 versions before and add these as affected?

Reply:
I'm not sure. Maybe just put them all? Or back to 2.0.0

Reply:
Yeah, I am not sure as well. I checked it back to 2.0.0 and I added this to the affected versions.